Source/Models/BrickDimension.swift

```diff
@@ -18,39 +18,106 @@ public struct BrickSize {
     }
 }
 
+public typealias RangeDimensionPair = (dimension: BrickDimension, minimumLength: CGFloat)
```

Comment:
I was uncertain about this one for a while, too. I wanted this to be applicable for height-based dimensions, too. However, I noticed in the code that we only support direct cases for height brick dimensions. I don't know if that's by design or a shortcoming. There's no good generic term that would encompass both width and height. For example, size is already reserved for a struct that contains both width and height.